### ISSUE TITLE
Refactor Wizard

### ISSUE THREAD

**Issue by atilaajones** [GraphFilter/GraphFilter#434]
Refactor the wiki following a structure according to the current version of the software. See the article [here](https://drive.google.com/file/d/1-zlOqiy0GmR-JrJHXzG8yERkZXuBOUrW/view?usp=drive_link).

Organize into the following topics (**suggestion**):

Software functionality

- Welcome window
- Open Function
- [Creating a New Project](https://github.com/GraphFilter/GraphFilter/wiki/User-Guide#creating-a-new-project)
- [Project Window](https://github.com/GraphFilter/GraphFilter/wiki/User-Guide#project-window)
  - Read and write graphs (previously [Tree File](https://github.com/GraphFilter/GraphFilter/wiki/User-Guide#tree-file))
  - [Graph Editing Features](https://github.com/GraphFilter/GraphFilter/wiki/Features#graph-editing-features)
  - New graph
  - Exports
- Filtering and Find example
  - Equations page
  - Conditions page
  - files page
  - review page
  - after filtering
- [Dictionary](https://github.com/GraphFilter/GraphFilter/wiki/Dictionary) (keep)
- [Run Project from source](https://github.com/GraphFilter/GraphFilter/wiki/Run-project-from-source-code) (keep)

In each topic explain very briefly what needs to be done. What is already written in the wiki, just reorganize. Always link to the Dictionary (which does not need to be changed).

In the Project window topic, you can explain how to get to this window.

**Comment by LBeghini** [comment]
Actually, I would like to propose something a bit different. 

- Home
  - About
  - History
  - Contributors
  - Acknowledgments
  - Publications
- User Guide
  - Quick guide (is the Getting Started from README.md)
  - Creating a new project
  - Read and write graphs
  - Graph Editing Features
  - New graph
  - Exports
  - Filtering
  - Find example
  - Equations
  - Conditions
  - Input graphs
  - What now? (after filtering)
- Dictionary
  - Invariants
    - Numeric
    - Boolean
    - Other
  - Template Graphs
  - Operations

> **Warning**
> The "Run project from source" should be completely removed from Wiki and added to a .md file inside the project structure (or even the README). And the "Getting started" should be removed from README.


The idea here is that the README file should be something more focused on developers, while the Wiki should be dedicated to the end users. Separating the objectives of the two documentations, we will not have to worry about updating two places with the same information (which is a recurring problem).

**Comment by fsoupimenta** [comment]
I believe that the User guide part could follow a structure like this:

- User Guide
  - Getting started 
  - Filtering
    - Find example
    - Equations
    - Conditions
    - Input graphs
    - Review
    - What now? (after filtering)
  - Project Window
    - Graph Editing Features
    - New Graph
    - Invariants
    - Exports

Getting started will show the welcome window and talk about the options, in the case of open, it already redirects to the `project window` section, and in the case of `new project` it already shows the method page guiding to filtering or to the `project window` again in the case of the blank project.

**Comment by atilaajones** [comment]
I agree with this structure. The structure of @fsoupimenta is more objective, but I suggest reversing the orders.

- User Guide
  - Getting Started
  - Main Window
    -  ...
    -  ...
  - Creating a Project
    - ...
    - ...

I also agree with the idea of @LBeghini: organize the Readme for developers, while the Wiki for users. But I think the beginning of the Readme should be kept: general text about the software and a link to the wiki. After that, the part for developers should begin. I agree with removing "Run project from source" from the Wiki.

What do you think?

**Comment by fsoupimenta** [comment]
@atilaajones 

First, just to make it clear that the idea is to combine the two structures (the side bar that @LBeghini suggested with the `User Guide` part in my comment).

We don't agree with reversing the order, as the idea would be to follow the main flow of the software, imagining that it would be easier for a new user to find their own way around. In addition, it was the flow used in the [WSGT](http://spectralgraphtheory.org) presentation that was very good, so we can even simulate a research using this structure, going into more detail about exporting to excel in the last topic and things like that.

**Comment by atilaajones** [comment]
Ah yes, then I agree. I hadn't noticed that the structure of **User Guide** was aligned with the sidebar, part of your comment.

In relation to changing the order, you're right, we'll follow the order that was presented at WSGT.

Then we can proceed with the reorganization via your posts.
